doc/langdef.md

```diff
@@ -950,7 +1269,7 @@ descriptions need to be updated in the code.
       google.protobuf.Timestamp.(string) -> int
     </td>
     <td>
-      get day of month from the date with timezone, one-based indexing
+      get day of month from the date with timezone, zero-based indexing
```

Comment:
This and the below timezone-related changes were not part of pull #11, so I think they're part of a third_party change that hadn't been synced to GitHub yet, so don't revert these changes.